src/schema-gen/TESTING.md

```diff
@@ -0,0 +1,27 @@
+# Schema generation testing
+
+Currently we are producing two schema language variant expressions of metaschema constraints, and planning a third:
+
+- XSD
+- JSON Schema, v7
+- (planned) Schematron
```

Comment:
Consider documenting plans as issues, not in documentation. IMHO, documentation should explain what currently exists.

Reply:
Removed from latest.